Earthfile

```
@@ -552,25 +552,32 @@ ci-release:
 
 # dind builds both the alpine and ubuntu dind containers for earthly
 dind:
-    ARG OS_IMAGE # e.g. alpine, ubuntu
-    ARG OS_VERSION # e.g. 3.18.0, 23.04
+    ARG --required OS_IMAGE # e.g. alpine, ubuntu
+    ARG --required OS_VERSION # e.g. 3.18.0, 23.04
+    ARG --required DOCKER_VERSION # e.g. 20.10.14
```

For `earthly doc` sake, I think it might be useful to move these comments above:
```suggestion
    # OS_IMAGE is the base image to use, e.g. alpine, ubuntu
    ARG --required OS_IMAGE
    # OS_VERSION is the version of the base OS to use, e.g. 3.18.0, 23.04
    ARG --required OS_VERSION
    # DOCKER_VERSION is the version of docker to use, e.g. 20.10.14
    ARG --required DOCKER_VERSION
```

---

good point!